<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="lab_tips1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_inputname_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exitsys">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_inputname">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="le_inputname_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>260</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_inputname_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>260</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips1_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>111</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_reg">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips2_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_inputname_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>260</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_tips2_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_isAdmin">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>超级权限</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;=</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
